### PR TITLE
Exporting missing types

### PR DESCRIPTION
**Pull request by skyka13711**
These types are needed to describe function signatures that wrap matrix-sdk methods

---
This PR currently has no changelog labels, so will not be included in changelogs.

A reviewer can add one of: `T-Deprecation`, `T-Enhancement`, `T-Defect`, `T-Task` to indicate what type of change this is, or add `Type: [enhancement/defect/task]` to the description and I'll add them for you.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -31,3 +31,9 @@ try {
 
 export * from "./matrix";
 export default matrixcs;
+
+export { EventType } from './@types/event';
```
Comment (turt2live) [inline src/index.ts:35]: we should probably be using `export *` for all of these.

**Review by turt2live — CHANGES_REQUESTED** [review]
Thanks for the contribution! It would be great if we can get a more complete title for this PR so it shows up in the changelog correctly, and it looks like the `yarn.lock` was accidentally committed with no relevant `package.json` changes.

**Comment by skyka13711** [comment]
fixed

**Review by turt2live — APPROVED** [review]
Thanks!

**Pull request merged**